### ISSUE TITLE
Question: Summary of codes found

### ISSUE THREAD

**Issue by leon1995** [charliermarsh/ruff#4114]
is there a way to just get the list of codes that were found without duplication?
e.g. if there are multiple errors found of the same code, the code will only be contained in the summary one time. Such that you get an overview of what codes are found.
```python
print('hello world 1')
print('hello world 2')
```
will print out the codes `T201` and `Q000` 2 times, but what I want is something like
```
Found the following errors:
- T201: 2 times
- Q000: 2 times
```
I have a project where ruff found more than 12000 errors and I want to discuss whether the codes are relevant for it or not. Therefore, a list of found codes with the frequency would be nice

**Comment by charliermarsh** [comment]
You should be able to run with the `--statistics` flag to get something very similar! Let me know if that doesn't solve your problem :)

**Comment by leon1995** [comment]
yes perfect. maybe you add it to [Usage)[https://beta.ruff.rs/docs/usage/] to increase the awareness of it :)

**Comment by calumy** [comment]
`--statistics` is currently covered in the [CLI ](https://beta.ruff.rs/docs/configuration/#command-line-interface) section of [Configuring Ruff](https://beta.ruff.rs/docs/configuration/).

**Comment by leon1995** [comment]
ok I completely overlooked it. thanks!
